README.md

```diff
@@ -36,7 +36,7 @@ Break this file up into two separate files:
 Here's `app/assets/javascripts/i18n/translations.js.i18njs`:
 
 ```yaml
----
+{}
```

Comment:
Yeah I would have figured the same, but it returns `nil` when the file is read in. I had to give it the empty brackets `{}` before it read an empty hash out. Alternatively I could alter the (file reader)[https://github.com/camertron/i18n-js-assets/blob/master/lib/i18n-js-assets/i18njs_file.rb#L9] to return an empty hash if it gets `nil` out of the file. What do you think?